securedrop/tests/functional/functional_test.py

```diff
@@ -25,6 +28,18 @@
 LOG_DIR = abspath(join(dirname(realpath(__file__)), '..', 'log'))
 
 
+# https://stackoverflow.com/a/34795883/837471
+class alert_is_not_present(object):
+    """ Expect an alert to not to be present."""
```

Comment:
Nit: `""" Expect an alert to not to be present."""` -> `"""Expect an alert to not be present."""`